### PR TITLE
Added SLALIB v2.5 sites from obs.f

### PR DESCRIPTION
**Pull request by luojing1211**
EDIT: Close #8 

The Pull request is based on #8 and fixed conflicts. 
There are some duplicated observatories such as "gbt" and "vla". Should we use the more accurate ones or just keep them both?

### CODE REVIEW

**Comment by pllim** [comment]
`+1,079` with duplicates... I am not sure about this, deferring to others. o_O

Or maybe we can split this PR into two -- One with *new* stuff only, one with *duplicates*

File: coordinates/sites.json [hunk 1]
```diff
@@ -746,5 +746,1084 @@
...
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 50.524611111217077,
+        "elevation_unit": "meter",
+        "longitude": 6.883750000000000,
+        "aliases": [
+            "Effelsberg 100 metre"
+        ]
+    },
+    "GBT": {
```
Comment (luojing1211) [inline coordinates/sites.json:1404]: This entry is the same observatory with line 675 but has a more accurate coordinate. In my opinion, we should keep the more accurate one.

**Comment by luojing1211** [comment]
@pllim, I removed two duplicated observatories, GBT and VLA. Could you review this PR?

**Comment by luojing1211** [comment]
@pllim Is this ready to merge?

**Comment by bmorris3** [comment]
I'm with @pllim – I'm not sure we should change observatory positions. Perhaps you can change this PR to be only additions?

**Comment by pllim** [comment]
As per https://github.com/astropy/astropy-data/pull/63#issuecomment-555663249 , I am closing this without merge. If you are still interested to pursue, please break down your changes into smaller PRs and avoid creating duplicates or modifying unrelated data. Thanks for your understanding and patience!

**Pull request closed without merging**